sw-precache-config.js

```diff
@@ -0,0 +1,8 @@
+module.exports = {
+  runtimeCaching: [
+    {
+      urlPattern: /^https?:\/\/(.*?).cloudfront.net\/bundles\/(.*?).js$/,
+      handler: 'networkFirst'
```

Comment:
If Netlify does not republish another version of the file in case it did not change, `cacheFirst` might be better.